src/components/Board.js

```diff
@@ -5,29 +5,84 @@ import axios from 'axios';
 import './Board.css';
 import Card from './Card';
 import NewCardForm from './NewCardForm';
-import CARD_DATA from '../data/card-data.json';
+// import CARD_DATA from '../data/card-data.json';
 
 class Board extends Component {
-  constructor() {
-    super();
-
+  constructor(props) {
+    super(props);
+    const {url, boardName} = this.props;
     this.state = {
       cards: [],
+      error: [],
+      baseUrl: url,
+      boardName: boardName,
+      deleteCardUrl: "https://inspiration-board.herokuapp.com/cards/"
     };
   }
 
+  componentDidMount(){
+    this.getCards();
+  }
+
+  getCards = () => {
+    axios.get(this.state.baseUrl + this.state.boardName + "/cards")
+    .then((response) => {
+      this.setState({ cards: response.data });
+    })
+    .catch((error) => {
+      this.setState({ error: error.message });
+    });
+  }
+
+  deleteCardCallback = (cardID) => {
+    axios.delete(this.state.deleteCardUrl + cardID)
+    .then((response) => {
+      console.log(`deleted ${response.data}`)
+      this.getCards();
+    })
+    .catch((error) => {
+      this.setState({ error: error.message });
+    });
+  }
+
+  submitNewCard = (newMessage) => {
+    axios.post(this.state.baseUrl + this.state.boardName + "/cards" + "?text=" + newMessage.text + "&emoji=" + newMessage.emoji)
+    .then((response) => {
+      console.log(`sent ${response.data}`)
+      this.getCards();
```

Comment:
Rather than calling `getCards` and doing another roundtrip GET request, you could modify the state directly here adding the new card to the list.